src/radix_connect/wallet_interaction/dapp_wallet_interaction/dapp_to_wallet/interaction_items/transaction/transaction.rs

```diff
@@ -14,28 +14,24 @@ impl DappToWalletInteractionTransactionItems {
 #[derive(Debug, Clone, Deserialize, Serialize, PartialEq, uniffi::Record)]
 #[serde(rename_all = "camelCase")]
 pub struct DappToWalletInteractionSendTransactionItem {
-    pub transaction_manifest: String,
+    #[serde(flatten, with = "UnvalidatedTransactionManifest")]
+    pub unvalidated_manifest: UnvalidatedTransactionManifest,
```

Comment:
I think all types doing custom Serde stuff like this ought to have a small JSON unit test here in this file.